### PR TITLE
Feature/mkdocs

### PR DESCRIPTION
**Pull request by JesusMcCloud**
Proper documentation using Material for MkDocs.
Each module has it's own dedicated section, there's a feature matrix, the Changelog is added automatically and the API docs are also linked. Examples are still missing, but there's an issue for that.

Please skim for obvious typos.

To test locally, install [material for mkdocs](https://squidfunk.github.io/mkdocs-material/) and either:
* `./gradlew mkDocsPrepare`, then execute `mkdocs serve` inside the _ docs_ folder to poke around interactively with auto-reloading, so you directly see any changes. or:
* `./gradlew mkDocsBuild` (which could fail, if some hyperlinks are not correct, but i will have to update those anyways before a release)

This is what it looks like:

## Main
image

## Supreme Section
image

## Feature Matrix
image

### CODE REVIEW

**Review by nodh — APPROVED** [review]
tl;dr ... pictures look fine

**Comment by JesusMcCloud** [comment]
@iaik-jheher VC-K needs a new Signum release to stabilize things, so I am squashing and merging this. We can fix errors and typos as we find them. At least we're in good company, if your code examples don't compile ;-)
